src/main/resources/db/migration/V20170713134558340__create_item_sequence.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER //
+CREATE PROCEDURE CreateSequence (`name` VARCHAR(30), `start` INT, `inc` INT)
+  BEGIN
+    CREATE TABLE IF NOT EXISTS `_sequences`
```

Comment:
Interesting. Why not just create the table outside this procedure since you know you're creating a sequence in a couple lines. Wrapping the table create and insert into a procedure seems overkill-ish.